lab-cortney/test/read-test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const index = require(__dirname + '/../index');
```

Comment:
Looks like you're trying to require Index, but Index doesn't module.exports anything!